### PR TITLE
Updating Identity Engine > Embedded Authentication content for Node.js SDK Password and Email factor use case

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Adding draft of the [Sign in with password and email factors](https://developer.okta.com/docs/guides/oie-embedded-sdk-use-cases/android/oie-embedded-sdk-use-case-sign-in-pwd-email/) use case for the Node.js language stack selector.
- **Is this PR related to a Monolith release?** n/a

### Resolves:

* [OKTA-408181](https://oktainc.atlassian.net/browse/OKTA-408181)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-email/nodejs/integrationsteps.md [hunk 1]
```diff
@@ -1 +1,66 @@
-Node.js sample code for this use case is coming soon. See [Okta Auth JS SDK](https://github.com/okta/okta-auth-js/blob/master/README.md) readme for further information.
\ No newline at end of file
+### Step 1: Build a sign-in page on the client and authenticate user credentials
+
+Build a sign-in page that captures both the user’s name and
+password. An example is shown below:
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-email/nodejs/integrationsteps.md:4]:
```suggestion
password, as shown in the following example:
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-email/nodejs/integrationsteps.md [hunk 2]
```diff
@@ -1 +1,66 @@
...
+
+<div class="common-image-format">
+
+![Sign-in page](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-sign-on-screenshot-sign-in.png
+ "Sign-in page")
+
+</div>
+
+When the user initiates the sign-in process, your app needs to create a new `OktaAuth` object and set its `username` and `password` properties to the values entered by the user. Send this object to the `idx.authenticate` method to start the authentication process. See [idx.Authenticate](https://github.com/okta/okta-auth-js/blob/master/docs/idx.md#idxauthenticate) for more information. This call begins the multi-factor authentication flow.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-email/nodejs/integrationsteps.md:13]:
```suggestion
When the user initiates the sign-in process, your app needs to create a new `OktaAuth` object and set its `username` and `password` properties to the values entered by the user. Send this object to the [`idx.authenticate`](https://github.com/okta/okta-auth-js/blob/master/docs/idx.md#idxauthenticate) method to start the authentication process. This call begins the multi-factor authentication flow. 
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-email/nodejs/integrationsteps.md [hunk 3]
```diff
@@ -1 +1,66 @@
...
+<div class="common-image-format">
+
+![Sign-in page](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-sign-on-screenshot-sign-in.png
+ "Sign-in page")
+
+</div>
+
+When the user initiates the sign-in process, your app needs to create a new `OktaAuth` object and set its `username` and `password` properties to the values entered by the user. Send this object to the `idx.authenticate` method to start the authentication process. See [idx.Authenticate](https://github.com/okta/okta-auth-js/blob/master/docs/idx.md#idxauthenticate) for more information. This call begins the multi-factor authentication flow. 
+
+If the username and password are valid, `idx.authenticate` should return a status of `Idx.Status:PENDING`. This status indicates that an additional factor needs to be verified before the sign in. In addition to the status, the `nextStep` field, included in the response, identifies the input parameters of the next step, in this case an authenticator of type email:
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-email/nodejs/integrationsteps.md:15]:
```suggestion
If the username and password are valid, `idx.authenticate` should return a status of `Idx.Status:PENDING`. This status indicates that an additional factor needs to be verified before the sign-in. In addition to the status, the `nextStep` field that is included in the response identifies the input parameters of the next step, which is, in this case, an authenticator of type email:
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-email/nodejs/integrationsteps.md [hunk 4]
```diff
@@ -1 +1,66 @@
...
+<div class="common-image-format">
+
+![Email verify](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-sign-in-pwd-email-screen-verify.png
+ "Email verify")
+
+</div>
+
+When the user selects the **email** factor, call `idx.authenticate` and pass in the authentication email authentication type, `({ authenticator: 'email' })`. With this call, Okta sends a verification code to the user's email.
+
+If the call is successful, the method should return a status of `Idx.Status:PENDING`, which indicates that the SDK is ready for the verification code. The `nextStep` field identifies the input parameter of the next step, in this case a verification code:
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-email/nodejs/integrationsteps.md:40]:
```suggestion
If the call is successful, the method should return a status of `Idx.Status:PENDING`, which indicates that the SDK is ready for the verification code. The `nextStep` field identifies the input parameter of the next step, which is a verification code in this case:
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-email/nodejs/integrationsteps.md [hunk 5]
```diff
@@ -1 +1,66 @@
...
+If not already done, build the email verification code page that accepts the code from the email.
+
+<div class="common-image-format">
+
+![Email verify](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-self-serv-screen-verify-email-code.png
+ "Email verify")
+
+</div>
+
+The next step is to call `idx.authenticate` and pass in the email verification code `({ verificationCode: 'xxx' })`. The next step is to handle the response from the call. If the email code was valid, the method should return a status of `Idx.Status:SUCCESS` and tokens. This status signifies that there are no more factors waiting to be enrolled and verified. If the steps described in [Set up your Okta org (for multi-factor use cases)](/docs/guides/oie-embedded-common-org-setup/aspnet/main/#set-up-your-okta-org-for-multifactor-use-cases) were properly followed, the user has successfully signed in and should be sent to the default sign-in home page.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-email/nodejs/integrationsteps.md:62]:
```suggestion
The next step is to call `idx.authenticate` and pass in the email verification code `({ verificationCode: 'xxx' })`. The next step is to handle the response from the call. If the email code is valid, the method should return a status of `Idx.Status:SUCCESS` and tokens. This status signifies that there are no more factors that are waiting to be enrolled and verified. If the steps in [Set up your Okta org (for multi-factor use cases)](/docs/guides/oie-embedded-common-org-setup/aspnet/main/#set-up-your-okta-org-for-multifactor-use-cases) were properly followed, the user has successfully signed in and should be sent to the default sign-in home page.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-email/nodejs/summaryofsteps.md [hunk 6]
```diff
@@ -1 +1,12 @@
 ## Summary of steps
+
+The sequence of steps for the sign in with password and email flow is shown below.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-email/nodejs/summaryofsteps.md:3]:
```suggestion
The following shows the sequence of steps for the sign-in with password and email flow.
```

**Review by christinechoy-okta — APPROVED** [review]
Just a few nits.

**Review by shuowu-okta — APPROVED** [review]

**Pull request merged**